frontend/app/components/Spots/SpotsList/SpotListItem.tsx

```diff
@@ -119,13 +120,13 @@ function SpotListItem({ spot, onRename, onDelete, onVideo, onSelect }: ISpotList
       {isEdit ? (
         <EditItemModal onSave={onSave} onClose={() => setIsEdit(false)} itemName={spot.title} />
       ) : null}
-      <div className='relative group overflow-hidden bg-black' style={{ width: '100%', height: 180 }}>
+      <div className='relative group overflow-hidden' style={{ width: '100%', height: 180, backgroundImage: `url('../../../assets/img/spotThumbBg.svg')`, backgroundSize: 'cover', backgroundPosition: 'center' }}>
```

Comment:
relative paths are not very good. What if we move this file elsewhere? Then relative path will break